src/common/motech-list/list.controller.js

```diff
@@ -17,6 +17,7 @@
     controller.$inject = ['$scope'];
     function controller($scope){
         var ctrl = this;
+        $scope.emptyListMessage = "Nothing Found";
```

Comment:
@PJSosnowski Could we define this sting in messages file so it could be easy translated?